tests/virt_autotest/login_console.pm

```diff
@@ -103,6 +103,7 @@ sub login_to_console {
     }
 
     if (!get_var("reboot_for_upgrade_step")) {
+        set_var("first_reboot_after_upgrade", "no") if (check_var("first_reboot_after_upgrade", "yes"));
```

Comment:
Will the  double_check_xen_role be still done for xen guest installation tests' login_console, whose var first_reboot_after_upgrade is '' initially? From https://openqa.suse.de/tests/8200504/logfile?filename=autoinst-log.txt, it seems it is skipped.

Reply:
Updated. 
```
double_check_xen_role if (is_xen_host and (!check_var("first_reboot_after_upgrade", "yes")));
```

Reply:
Verification also done: https://openqa.suse.de/tests/8202762#step/login_console/10